setup.py

```diff
@@ -44,12 +44,12 @@
         'future',
         'backports.csv',
         'keyring',
-        'keyrings.alt'
     ],
     extras_require = {
         'pandas':  ["pandas"],
         'pysftp': ["pysftp>=0.2.8"],
         'boto3' : ["boto3"],
+        ':sys_platform=="linux2" or sys_platform=="linux"': ['secretstorage<3.0.0', 'keyrings.alt'],
```

Comment:
why do we need to categorize `linux` and `linux2`? Can we combine them into `linux` or OS specific?

Reply:
Please add a comment mention the reason why we have to do this. If this is temporarily, please make sure that it's clear indicate what should happen instead.

Reply:
From my understanding of the documentation, 'linux' is returned in Python 3. 'linux2' is returned in Python 2. https://docs.python.org/2/library/sys.html#sys.platform

we can't use the recommended `startswith('linux')` because sys_platform is not a variable. it is a magics string that is bound to a `lambda: sys.platform`. if we try to use `startswith` we get an error `Invalid environment marker: sys_platform.startswith("linux")`